src/hotspot/share/runtime/thread.cpp

```diff
@@ -3658,6 +3658,19 @@ static void call_initPhase3(TRAPS) {
                                          vmSymbols::void_method_signature(), CHECK);
 }
 
+Method* findTestMethod(InstanceKlass *klass) {
+    int n_methods = klass->methods()->length();
+    for (int i = 0; i < n_methods; ++i) {
+        Symbol *name = klass->methods()->at(i)->name();
+        char cname[100];
+        name->as_C_string(cname, 100);
```

Comment:
NB. Another ways to to this 
1.
```
{ 
ResourceMark rm; // create a mark on thread local "resource buffer". All values in the buffer (created under single mark) are freed by ResourceMark destructor. 
const char* name = name->as_C_string();  // will copy C string to resource buffer. The value is valid only while in this block (because of ResrouceMark)
}
```
2. `Symbol` have `equals` method, that allows compare without intermediate copies